reframe/frontend/executors/policies.py

```diff
@@ -304,4 +299,4 @@ def exit(self):
                 raise
 
         self.printer.separator('short single line',
-                               'all spawned checks have finished')
+                               'all spawned checks have finished\n')
```

Comment:
If the new line is to accommodate the retries feature, it should not be added here.

Reply:
I think it should in general be here, independent of the retries feature.

Reply:
No, it's on purpose without a newline.

Reply:
Please remove the extra newline, because it keeps consistency of the output in exceptional cases (cancellation, serial execution etc.)

Reply:
Done.